requirements.txt

```diff
@@ -1,2 +1 @@
-gnureadline
```

Comment:
Keep this for upstream compatibility. Dennis modified our Conda recipe to remove the line.